### ISSUE TITLE
Throttling options don't work example /iorate:1m comes up as an Invalid Paramater

### ISSUE THREAD

**Issue by VaibhavMittal** [MicrosoftDocs/windowsserverdocs#7744]
Throttling options don't work example /iorate:1m comes up as an Invalid Paramater

What's the problem here?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: bf2281d4-1470-0658-ec5a-687fe2cd451c
* Version Independent ID: 2ce2be3e-e3a2-790c-3bff-90efe7318450
* Content: [Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy#related-links)
* Content Source: [WindowsServerDocs/administration/windows-commands/robocopy.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/administration/windows-commands/robocopy.md)
* Service: **windows-server**
* Sub-service: **administration**
* GitHub Login: @Xelu86
* Microsoft Alias: **wscontent**

**Comment by Xelu86** [comment]
Hey @VaibhavMittal,

Thanks again for contacting Microsoft, we value your feedback.

We don't know the full command you're running. We also addressed this in #7547. The documentation is correct. If you're having issue's running certain commands, we recommend reaching out to our [Microsoft Support Community](https://answers.microsoft.com/).

Here is an example copying the same data in speeds of 1k/1m/1g. You'll notice the transfer time & speed difference in each output:

```
robocopy C:\Records D:\Backup /iorate:1k
```
<img width="1042" alt="image" src="https://github.com/MicrosoftDocs/windowsserverdocs/assets/103963494/a71e6a83-6511-46d6-96f4-968d28f4e767">

```
robocopy C:\Records D:\Backup /iorate:1m
```
<img width="1045" alt="image" src="https://github.com/MicrosoftDocs/windowsserverdocs/assets/103963494/6fa2edec-51cd-4766-9638-6ba299a90e2a">

```
robocopy C:\Records D:\Backup /iorate:1g
```
<img width="1042" alt="image" src="https://github.com/MicrosoftDocs/windowsserverdocs/assets/103963494/0cb3163a-f552-469b-b2ed-ef6880533f89">


As this is a non-documentation issue, we'll now be closing this thread.

Thank you and have a most pleasant day.
#please-close

**Comment by lcringboy** [comment]
The iorate parameter supported by robocopy must be used in Windows Server 2022 or later.
